mithril-explorer/package-lock.json

```diff
@@ -36,7 +36,7 @@
     },
     "../mithril-client-wasm/pkg": {
       "name": "mithril-client-wasm",
-      "version": "0.6.0",
+      "version": "0.5.2",
```

Comment:
Oups, this is not the correct version number (that should be `0.6.1`).

Having `npm install` use the correct version  ss a little tricky with projects that depends on the wasm library as it use the version of the autogenerated `mithril-client-wasm/pkg/package.json`.
So to reflect the `Cargo.toml` version you can either rebuild the wasm library or update `mithril-client-wasm/pkg/package.json` manually, then rerun `npm install`.

Reply:
Done